pkg/cmd/cli/cmd/buildlogs.go

```diff
@@ -17,11 +17,14 @@ import (
 )
 
 const (
-	buildLogsLong = `Retrieve logs from the containers where the build occurred.
+	buildLogsLong = `
+Retrieve logs for a build
 
-NOTE: This command may be moved in the future.`
```

Comment:
I don't know.  Maybe?  Maybe not?

On Mon, Jul 13, 2015 at 7:40 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/buildlogs.go
> <https://github.com/openshift/origin/pull/3669#discussion_r34454781>:
>
> >
> > -NOTE: This command may be moved in the future.`
>
> Unrelated to this, but are we going to make an "oc build " branch to
> eventually retire these top level pieces (start and cancel come to mine)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3669/files#r34454781>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift